data/data/google/network/firewall.tf

```diff
@@ -0,0 +1,19 @@
+resource "google_compute_firewall" "global" {
```

Comment:
I think it would be vest if this matches the https://github.com/openshift/installer/blob/master/data/data/aws/vpc/sg-master.tf and https://github.com/openshift/installer/blob/master/data/data/aws/vpc/sg-worker.tf

Reply:
@abhinavdahiya Are we matching the naming of masters and workers, or the way aws is specifying ports?

Reply:
splitting into 2 different sets of `control-plane and compute` is probably useful like AWS.

Also making sure our firewall for GCP looks like AWS in terms of what's allowed and what's not within VPC, ingress from outside and egress to outside.

Reply:
@abhinavdahiya PTAL